### PR TITLE
Brings back the Disco Inferno shuttle, but as an admin-only shuttle

### PR DESCRIPTION
**Pull request by ATH1909**
## About The Pull Request

This is literally just a reversion of https://github.com/tgstation/tgstation/pull/53982 plus a one line change.

Said one-line change sets can_be_bought to FALSE for the Disco Inferno shuttle, making it an admin-only shuttle (like Oh hi, Daniel).

## Why It's Good For The Game

image

Rohesie themself said in their Disco Inferno removal PR that Disco Inferno should be an admin event shuttle. When asked why they didn't just make the one line change made by this PR to make Disco Inferno admin-only, Rohesie said:

image

By that logic, every admin-only shuttle should be removed from the game, because they all somehow "limit creativity". Mate, do you really think that every admin's going to want to spend 20 minutes manually building a shuttle (instead of adminning) every time they want to send the crew a special shuttle/someone tries to TC trade for a removed shuttle? Admins have so far shown restraint with other admin-only shuttles (like Oh hi, Daniel); they won't spam Disco Inferno every round just because they can.

tldr;
Disco Inferno did NOT deserve to be deleted from the code entirely. This PR adds it back.

## Changelog
:cl:
add: The Disco Inferno emergency escape shuttle is back, but it's admin-only now.
/:cl:

### CODE REVIEW

**Comment by Rohesie** [comment]
Up to the other maintainers if someone wants to merge this. I'd rather we removed the grief shuttles in general.

**Comment by moocowswag** [comment]
Maybe make it purchase-able/free if you emag the command console? Disco inferno is sought after by tators that have access to environmental protection and want to hijack.

**Comment by Jared-Fogle** [comment]
I'll merge if #54012 goes stale.
